src/OnlineRetailPortal.Contracts/Contracts/IImageHandler.cs

```diff
@@ -0,0 +1,25 @@
+﻿using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace OnlineRetailPortal.Contracts.Contracts
+{
+    public interface IImageHandler
+    {
+        /// <summary>
+        /// Takes an UploadImageRequest type object and tries to save the image in it
+        /// </summary>
+        /// <param name="request"></param>
+        /// <returns></returns>
+        Task<UploadImageResponse> UploadImage(UploadImageRequest request);
```

Comment:
async methods should be suffixed with `async`

Reply:
Okay, Done